src/hotspot/share/cds/metaspaceShared.cpp

```diff
@@ -1353,6 +1353,8 @@ MapArchiveResult MetaspaceShared::map_archives(FileMapInfo* static_mapinfo, File
     log_info(cds)("initial optimized module handling: %s", CDSConfig::is_using_optimized_module_handling() ? "enabled" : "disabled");
     log_info(cds)("initial full module graph: %s", CDSConfig::is_using_full_module_graph() ? "enabled" : "disabled");
   } else {
+    // The RW and RO regions are in a reserved space so they must only be
+    // unmapped when the reserved space is released
```

Comment:
I think this comment is not always true and should be removed. On Windows, sometimes the RO/RW regions are not in a ReservedSpace.